src/network/connectionpool.py

```diff
@@ -113,7 +118,7 @@ def removeConnection(self, connection):
                     pass
         connection.handle_close()
 
-    def getListeningIP(self):
+    def getListeningIP(self):       # pylint: disable=no-self-use
```

Comment:
this can probably be made into a `@staticmethod`

Reply:
Sometimes, @staticmethod breaks the code .So, that's why I disable the warning. 
But, will update and correct the code. :)

Reply:
Yes, sometimes it breaks, but I looked at it an in this case it's ok.